draft-ietf-rats-architecture.md

```diff
@@ -327,7 +327,11 @@ same way as any other Verifier), for evaluating these Attesters' trustworthiness
 Then the Lead Attesting Environment combines the Attestation Results into
 the final Evidence of the whole Composite Attester which is sent off to the remote
 Verifier, which might treat the claims obtained from the local Attestation Results
-as if they were Evidence.
+as if they were Evidence. In this situation, the internal Verifier may need to be
+trusted by the Endorser and Verifier Owner before getting the Endorsements
```

Comment:
This is confusing.   There are multiple Verifiers (local and remote) and thus there can be two Endorsers, and two Verifier Owners.  So "the Endorser" and "the Verifier Owner" is ambiguous.